library/src/main/java/org/codice/security/saml/MetadataConfigurationParser.java

```diff
@@ -30,6 +31,7 @@
 public class MetadataConfigurationParser {
```

Comment:
I will probably look into donating the changes to `MetadataConfigurationParser` back to DDF.

Reply:
You should 💯